openwisp_controller/config/base/vpn.py

```diff
@@ -661,4 +675,10 @@ def _auto_vxlan(self):
     def _auto_ip(self):
         if not self.vpn.subnet:
             return
+        if self.vpn.subnet.subnetdivisionrule_set.filter(
+            organization_id=self.config.device.organization
+        ).exists():
```

Comment:
please check that the rule is of type VPN, other rules should not interact with `VpnClient`